lib/malloc.cpp

```diff
@@ -269,6 +269,7 @@ namespace quda {
     return ptr;
   }
 
+#define HOST_ALLOC // this needs to be set presently on P9
```

Comment:
should this be defined by cmake?

Reply:
I don't think so.  Making it a cmake parameter means a global rebuild when changing it, where as for debugging purposes you really want to be able to make a local change and quick rebuild.  This isn't something we ever need to change, but I left the old code in there for future testing.

Reply:
Ok. Was not sure whether this should be enabled only on Power.